src/aavrutiDJun20/ExtractItemId.java

```diff
@@ -0,0 +1,31 @@
+package aavrutiDJun20;
+
+public class ExtractItemId {
+
+	//Extract ItemId
+	void findItemId(String input) {
+		System.out.println("Input  --> " + input);
+		if(input.contains("itemId")) {
+			String itemIdValue = input.substring(input.lastIndexOf("itemId="),input.lastIndexOf("/")).replace("itemId=", "");
+			if(itemIdValue.contains("&")) {
+				itemIdValue = itemIdValue.substring(0,itemIdValue.lastIndexOf("&"));
+			}
+			System.out.println("Output --> ItemId is " + itemIdValue);
+		}
```

Comment:
if (input.contains("itemId")) {
			int item = input.indexOf("itemId");
			String sub = input.substring(item);
			String s = sub.split("=")[1].split("&")[0].split("/")[0];
			System.out.println("ItemID is " + s);
		} else {
			System.out.println("No itemID present.");
		}

Suggesting an alternative You can also write : 
try{
String str = input.split("itemId=")[1].replace("/", "").split("&")[0];
System.out.println("Item id is " + str);

}catch(StringIndexOutOfBoundsException siob){
			System.out.println("Item ID -> Not present");
		}

This two line in try block and it will work.